pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -29,15 +29,15 @@ import (
 )
 
 const (
-	// maxAllowableNUMANodes specifies the maximum number of NUMA Nodes that
+	// defaultMaxAllowableNUMANodes specifies the maximum number of NUMA Nodes that
 	// the TopologyManager supports on the underlying machine.
 	//
 	// At present, having more than this number of NUMA Nodes will result in a
 	// state explosion when trying to enumerate possible NUMAAffinity masks and
 	// generate hints for them. As such, if more NUMA Nodes than this are
 	// present on a machine and the TopologyManager is enabled, an error will
 	// be returned and the TopologyManager will not be loaded.
-	maxAllowableNUMANodes = 8
+	DefaultMaxAllowableNUMANodes = 8
```

Comment:
Is there a reason to use a capital `D` to export this variable outside of the package?

Reply:
thanks! you are right, I was using a lowercase `d`, and then I referenced it in the test file, and then my vs-code reported an error and I thought I needed an uppercase `D`, and now that I've changed it back, everything is fine.